test/remote_reporter.js

```diff
@@ -81,6 +84,21 @@ describe('Remote Reporter should', () => {
         assert.isOk(LocalBackend.counterEquals(metrics.reporterDropped, 1));
     });
 
+    it ('should have coverage for simple code paths', () => {
+        let sender = new UDPSender();
+        let reporter = new RemoteReporter(sender);
+        assert.equal(reporter.name(), 'RemoteReporter');
+    });
+
+    it ('should set process without a sender', () => {
```

Comment:
don't follow this, there is still a sender, but without setProcess method, which makes little sense since the method is a requirement of the Sender API. Don't know why this test is needed in the first place, since remote reporter without sender is invalid combination, it can just throw exception from the constructor.